modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js

```diff
@@ -26,9 +27,9 @@ const TablePagination = ({
 }) => {
 	if (showDeltasDropDown || totalItems > itemsPerPage) {
 		const defaultLabels = {
-			paginationResults: 'Showing {0} to {1} of {2}',
-			perPageItems: '{0} items',
-			selectPerPageItems: '{0} items',
+			paginationResults: i18n.translate('showing-x-to-x-of-x'),
+			perPageItems: i18n.translate('x-items'),
+			selectPerPageItems: i18n.translate('x-items'),
```

Comment:
Hey @claricebrasil, just for clarify me, this is works integrate with `defaultLabels` for clay?

Reply:
I didn't see any problems, but actually is not making any difference in the layout, so I leave the previous version without the translate.